integreat_cms/cms/models/events/event.py

```diff
@@ -44,16 +45,18 @@ def filter_upcoming(self, from_date: datetime | None = None) -> EventQuerySet:
         :param from_date: The date which should be used for filtering, defaults to the current date
         :return: The Queryset of events after the given date
         """
-        from_date = from_date or timezone.now().date()
+        from_date = (from_date or timezone.now()).min.replace(
+            tzinfo=ZoneInfo(key="UTC")
+        )
         return self.filter(
-            Q(end__gte=from_date)
+            Q(end__date__gte=from_date)
```

Comment:
```suggestion
            Q(end__gte=from_date)
```
`end` is a datetime,`from_date` as well, so no need for `__date`.